be-python-flask/Jenkinsfile.template

```diff
@@ -20,6 +20,7 @@ odsPipeline(
   ]
 ) { context ->
   stageBuild(context)
+  stageScanForSonarqube(context)
```

Comment:
hi @stitakis , please, do not load a second time the `stageScanForSonarqube`, it was already existing in https://github.com/opendevstack/ods-quickstarters/blob/7e5f08e8dce7371fab79129d2429c86c5c6d7dee/be-python-flask/Jenkinsfile.template#L47 

the new implementation with the new test suite already in master (not added in 2.x due to breaking changes from the slave) has it more explicit.

Reply:
ok... I didn't expected the sq stage within stageBuild... anyway I reverted this change. So no change in flask qs

Reply:
please, instead of adding a new commit for removing an issue added on previous commit, just rebert the commit itself, so no need to keep this in the history.